From 16f8a8849621c0b5565fd570014f2b239998b017 Mon Sep 17 00:00:00 2001
From: Murala S Swamy <ssmurala@habana.ai>
Date: Mon, 01 Jul 2024 08:34:31 +0300
Subject: [PATCH] [FSW-6860] fs1: use new d2d dis efuse

with this patch, removed old d2d_en efuse usage
and included new efuse d2d_dis efuse bit as it
was updated in new efuse excel file v0p7.

Change-Id: Ie8508410593483ef54c19a1b863694c012103db0
---

diff --git a/arch/fs1/arc/dev_recovery.c b/arch/fs1/arc/dev_recovery.c
index 302aff4..fc22455 100644
--- a/arch/fs1/arc/dev_recovery.c
+++ b/arch/fs1/arc/dev_recovery.c
@@ -55,7 +55,7 @@
 	uint64_t start;
 	uint32_t val;
 
-	if (!(boot_rom_global.config2_fuse & EFUSE_SYSCFG2_D2D_ENABLE))
+	if (boot_rom_global.config2_fuse & EFUSE_SYSCFG2_D2D_DISABLE)
 		return; /* skip d2d init */
 
 	boot_rom_status_update(PROGRESS_BOOT_ROM_D2D_LINK_START);
